net/keepalived/files/usr/libexec/rpcd/keepalived

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /usr/share/libubox/jshn.sh
+
+RPC_SCRIPTS=/usr/libexec/keepalived/rpc
+
+[ -d $RPC_SCRIPTS ] && include $RPC_SCRIPTS
```

Comment:
I am currently trying to write a plugin for the ubus keepalived.
To verify your pull request.
Can you give me an example?
I'm not sure if the shell (openwrt uses ash) supports this `include` buildin word.

Reply:
Its default busysbox shell ash only.

`include` comes from `/lib/functions.sh`

Here are two examples of the keepalived rpcd plugin.

1. Valid RPC with a success response.
```sh
cat /usr/libexec/keepalived/rpc/example.sh 
#!/bin/sh

example() {
	json_init
	json_add_string key value
	json_dump
}

example_help() {
	json_add_object example
	json_close_object
}
```

2. Invalid RPC, partial RPC.
`touch /usr/libexec/keepalived/rpc/example2.sh`

3. Restart rpcd and list keepalived RPCs.
```sh
/etc/init.d/rpcd restart
ubus -v list keepalived
'keepalived' @19c942e2
	"dump":{}
	"example":{}
	"example2":{}
```

4. RPC call response
```sh
ubus call keepalived example
{
	"key": "value"
}
ubus call keepalived example2
{
	"error": "invalid call example2"
}
```